features/post/post-comment/post-comment.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { ReactNode } from 'react';
+
+import useRelativeTime from '@/shared/hooks/useRelativeTime';
+import { useTranslation } from '@/shared/hooks/useTranslation';
+import { Avatar, Typography } from '@/shared/ui';
+import clsx from 'clsx';
+
+import s from './post-comment.module.scss';
+
+export type PostCommentProps = {
+  avatar?: string;
+  children?: ReactNode;
+  childrenInfoSection?: ReactNode;
+  createdAt: string;
+  iconElement?: ReactNode;
+  likesCount: number;
+  text: string;
+  textWidth?: string;
+  userName: string;
+};
+
+export const PostComment = (props: PostCommentProps) => {
+  const {
+    avatar,
+    children,
+    childrenInfoSection,
+    createdAt,
+    iconElement,
+    likesCount,
+    text,
+    textWidth,
+    userName,
+  } = props;
+  const relativeTime = useRelativeTime(createdAt);
+  const { t } = useTranslation();
+
+  const getInitials = (userName: string): string => {
+    const words = userName.split(' ');
+    const initials = words.map(word => word.charAt(0).toUpperCase());
+
+    return initials.join('');
+  };
+
+  return (
+    <div className={s.root}>
+      <div className={s.comment}>
+        <Avatar fallback={getInitials(userName)} size={'small'} src={avatar} />
+        <div className={s.body}>
+          <Typography.Regular14 className={clsx(s.body__text, textWidth)}>
+            <Typography.Bold14>{userName}</Typography.Bold14>
+            {' ' + text}
+          </Typography.Regular14>
+          <div className={s.body__info}>
```

Comment:
Creating an expandable section for an embedded component is a good idea, but not fully realized.
The "answer" button can also be considered as part of the expandable section, since it is only present in the "private comment" implementation. And since it is on the same level as the relative date and number of likes, it makes sense to make a render prop to implement this idea:
```typescript
<PostComment
  bottomSection={(RelativeTime, LikesCount) => {
    return (
      <PostComment.BottomSectionWrapper>
        <RelativeTime />
        <LikesCount />

        <Button variant={'text'} className={s.answerButton} onClick={someStuff} />
      </PostComment.BottomSectionWrapper>
    )
  }}
/>
```
Note that the compound component `PostComment.BottomSectionWrapper` is used here, which is just a convenient styled wrapper for the options group (just like the `InputAdornment` component for `Input`). Passing components to render-prop will also greatly simplify their customization in more specific `PostComment` based components.